components/SearchForm/SearchForm.vue

```diff
@@ -12,7 +12,8 @@
         />
       </button>
     </div>
-    <el-button class="mr-8 btn-submit-search" @click="$emit('search')">
+    <el-button class="btn-submit-search" @click="$emit('search')">
+        <font-awesome-icon icon="search"/>
```

Comment:
We actually have a magnifying glass icon that you should be able to use here rather than importing font awesome. If you notice on the global search bar, we are using our custom magnifying icon. Something like:

```
<svg-icon
  icon="icon-magnifying-glass"
  height="16"
  width="16"
/>
```

Reply:
1. Good call, actually realized that once I saw the second search bar was using it
2. Don't apologize, that is definitely worth a call out. I thought I had done that, but I must have missed it. I need to update my editor formatting settings
3. Yes I will keep mobile in mind. That should work, I will play around with it and see what looks good